nautobot/extras/models/jobs.py

```diff
@@ -586,6 +586,32 @@ def __init__(self, *args, **kwargs):
     class Meta:
         ordering = ["-created"]
         get_latest_by = "created"
+        indexes = [
+            models.Index(
+                name="extras_jobresult_created_idx",
+                fields=["created"],
+            ),
+            models.Index(
+                name="extras_jobresult_rcreated_idx",
+                fields=["-created"],
+            ),
```

Comment:
Ignorant question - is there actually a need/value to indexing the same field in both directions like this? I would have naively assumed that one index would cover both directions. (and in that case, wouldn't adding `db_index=True` on the `created` field solve this more succinctly?)

Reply:
I would rather see the forward indexes done on the field using `db_index=True`, only leaving the reverse indexes here in the `Meta.indexes`.

Reply:
Sorry it took me a while to update. @glennmatthews @jathanism updated from feedback.